src/phasorpy/phasor.py

```diff
@@ -1113,6 +1116,28 @@ def phasor_calibrate(
             f'reference_real.shape={ref_re.shape} '
             f'!= reference_imag.shape{ref_im.shape}'
         )
+
+    skip_axis, axis = _parse_skip_axis(skip_axis, re.ndim)
+    if re.ndim > 0:
+        harmonic, has_harmonic_axis = parse_harmonic(harmonic, re.shape[0] * 2)
+    else:
+        has_harmonic_axis = False
+        harmonic = 1 if harmonic is None else harmonic
```

Comment:
This can be a good idea. I will try some modifications to `parse_harminc` and we can discuss if it can be generalized.

Reply:
OK. Since the `parse_harmonic` function is frequently used with either known `samples` or known `max(harmonic)`, we may just support both cases (in addition to "unknown"), without having to do the error prone conversion from `max(harmonic)` to `samples` in the code. Let me know if you are working on it otherwise I will.

Reply:
Great! Yes of course, go ahead, if you have the time of course no hurry. I think you  already have a better insight of this modification from what I've been trying.

Reply:
Implemented in #138. Let me know if that helps.

Reply:
> `parse_harmonic(harmonic, re.shape[0] * 2)`

As far as I understand this is not the correct check. The `harmonic` input can contain any harmonics. What needs to be verified is `harmonics, has_harmonic_axis = parse_harmonic(harmonic)` and then `if has_harmonic_axis and len(harmonic) != re.shape[0]: raise ValueError`. If `has_harmonic_axis` is false, there should be no harmonic axis present in `real` and `imag`.

Reply:
It looks like you'll have to special-handle the `harmonics='all'` case, which means that `real` and `imag` contain the `[1..real.shape[0]]` harmonics in the first axis.